docs/src/dev-docs/new-architecture.rst

```diff
@@ -71,9 +70,6 @@ The ``ModelInterface`` is the main model class and must implement a
             self.hypers = model_hypers
             self.dataset_info = dataset_info
 
-        def save_checkpoint(self, path: Union[str, Path]):
-            pass
-
         @classmethod
         def load_checkpoint(cls, path: Union[str, Path]) -> "ModelInterface":
             pass
```

Comment:
```suggestion
```

Reply:
This is still present for all architectures (we need a `model.load_checkpoint` for export, where there is no `Trainer`)

Reply:
But then I think we should also provide for consistency a  `save_checkpoint` for the model. 

Which means basically this PR adds a `save_checkpoint()` and `load_checkpoint` for the trainer.

Reply:
I don't think so. We should only have what's necessary. If people want to have it (and call it inside the Trainer.save_checkpoint) then it's up to them

Reply:
Yeah makes sense